pkg/detectors/npmtokenv2/npmtokenv2_test.go

```diff
@@ -1,7 +1,7 @@
 //go:build detectors
 // +build detectors
 
-package npmtoken_new
+package npmtokenv2
```

Comment:
nit: Linter mentions you have some undefined types. Likely need ot just change the name to use v2 instead of new?